wallets/metamask-cosmos-snap/package.json

```diff
@@ -0,0 +1,101 @@
+{
+  "name": "@cosmos-kit/leap-metamask-cosmos-snap",
+  "version": "0.16.11",
+  "description": "cosmos-kit wallet connector",
+  "author": "mayank.yadav@leapwallet.io",
+  "contributors": [
+    {
+      "name": "Mayank Yadav"
+    }
+  ],
+  "homepage": "https://github.com/cosmology-tech/cosmos-kit#readme",
+  "license": "SEE LICENSE IN LICENSE",
+  "main": "main/index.js",
+  "module": "module/index.js",
+  "typings": "types/index.d.ts",
+  "directories": {
+    "lib": "src",
+    "test": "__tests__"
+  },
+  "files": [
+    "types",
+    "main",
+    "module"
+  ],
+  "scripts": {
+    "build:main": "cross-env BABEL_ENV=production babel src --out-dir main --delete-dir-on-start --extensions \".tsx,.ts,.js\"",
+    "build:module": "cross-env MODULE=true babel src --out-dir module --delete-dir-on-start  --extensions \".tsx,.ts,.js\"",
+    "build": "npm run build:module && npm run build:main",
+    "build:ts": "tsc --project ./tsconfig.json",
+    "prepare": "npm run build",
+    "dev": "cross-env NODE_ENV=development babel-node src/index",
+    "lint": "eslint --ext .tsx,.ts .",
+    "format": "eslint --ext .tsx,.ts --fix .",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:debug": "node --inspect node_modules/.bin/jest --runInBand"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cosmology-tech/cosmos-kit"
+  },
+  "keywords": [
+    "cosmos-kit",
+    "cosmos",
+    "leap",
+    "wallet"
+  ],
+  "bugs": {
+    "url": "https://github.com/cosmology-tech/cosmos-kit/issues"
+  },
+  "jest": {
+    "testPathIgnorePatterns": [
+      "main/",
+      "module/",
+      "types/"
+    ]
+  },
+  "devDependencies": {
+    "@babel/cli": "7.18.10",
+    "@babel/core": "7.19.1",
+    "@babel/eslint-parser": "7.19.1",
```

Comment:
this has the old babel build stuff in it — we've upgraded to use `tsc` pure — can you redo this using the new stuff?

Reply:
Ok, I'll close this pr and create a another one with the new stuff.